frontend/package.json

```
@@ -24,7 +24,8 @@
     "react-hook-form": "7.24.0",
     "react-redux": "7.2.6",
     "react-router-dom": "6.2.1",
-    "react-scripts": "5.0.0"
+    "react-scripts": "5.0.0",
+    "react-select": "^5.2.2"
```

```suggestion
    "react-select": "5.2.2"
```
creteria [B19](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md)